gensrc/thrift/PlanNodes.thrift

```diff
@@ -213,8 +215,54 @@ struct TEsScanRange {
   4: required i32 shard_id
 }
 
-struct TFileScanRange {
+struct TFileTextScanRangeParams {
+    3: optional i8 column_separator;
+    4: optional i8 line_delimiter;
+    // for multibytes separators
+    5: optional i32 column_separator_length = 1;
+    6: optional i32 line_delimiter_length = 1;
+    7: optional string column_separator_str;
+    8: optional string line_delimiter_str;
+}
```

Comment:
does we support more options such as quote, escape, include header or not and so on?
suggest TFileTextScanRangeParams use map<k, v> struct, as that we could support more options without change the TFileTextScanRangeParams

Reply:
`map` is not an efficient way for serde.
And if we add more options in future, we still need to modify BE or FE code.
So I think it is ok for now.